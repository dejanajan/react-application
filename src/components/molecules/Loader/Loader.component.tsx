import React from "react";
import * as LoaderStyle from "./Loader.styles";

export function Loader() {
  return (
    <LoaderStyle.Loader>
      <p>Loading ...</p>
    </LoaderStyle.Loader>
  );
}

export default Loader