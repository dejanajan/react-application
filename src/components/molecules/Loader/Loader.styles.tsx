import styled from "styled-components";

export const Loader = styled.div`
  margin: 50px;
  font-size: 20px;
  color: red;
`;
