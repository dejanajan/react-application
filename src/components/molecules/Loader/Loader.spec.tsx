import React from "react";
import Loader from "./Loader.component";
import renderer from "react-test-renderer";


test("Loader", () => {
  const loader = renderer
    .create(
        <Loader />
    );

  const tree = loader.toJSON();  
  expect(tree).toMatchSnapshot();
});
