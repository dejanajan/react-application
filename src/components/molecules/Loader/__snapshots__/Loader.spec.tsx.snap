// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader 1`] = `
<div
  className="sc-AxjAm jnWFAI"
>
  <p>
    Loading ...
  </p>
</div>
`;
